app.yaml

```diff
@@ -1,8 +1,12 @@
 runtime: python27
-threadsafe: true
+threadsafe: false
 api_version: 1
 instance_class: F4
 
+automatic_scaling:
+  min_idle_instances: 1
```

Comment:
It means we will have a minimum one instance running at all time, rather than 0. Unsure that billing is a concern for this project?

Reply:
It is not a big concern.  Our GAE bill has been about $11/day.  It recently went up to $17/day right about the time that we started using Cloud NDB.  I'll keep an eye on new daily charges, but I expect that it won't be too much higher than what we are already paying.